recipes/pyamrex/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "pyamrex" %}
+{% set version = "23.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AMReX-Codes/pyamrex/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 44dd3ab467260ea797dec4fcb939599fad811e8ac598bdc0ab05da24369606fa
+  patches:
+    # Adds pip_install_nodeps target, part of 23.11+
+    #   https://github.com/AMReX-Codes/pyamrex/pull/203
+    - 203.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang         # [win]
+    - cmake
+    - lld           # [win]
+    - make          # [unix]
+    - ninja         # [win]
+    - libgomp       # [linux]
+    - llvm-openmp   # [osx or win]
+    - pkgconfig
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
+  host:
+    - amrex =={{ version }}
+    - numpy
+    - pip
+    - pybind11
+    - python
+    - setuptools
+    - wheel
+  run:
+    - amrex =={{ version }}
+    - {{ pin_compatible('numpy') }}
+    - python
+    - libcxx  # [osx]
+
+test:
+  requires:
+    - pytest
+  imports:
+    - amrex
+  source_files:
+    - tests/*
+  files:
+    - test.sh   # [unix]
+    - test.bat  # [win]
+  commands:
+    # smoke tests: execute
+    - ./test.sh      # [unix and build_platform == target_platform]
+    - call test.bat  # [win]
```

Comment:
If you rename these to `run_test.sh` and `run_test.bat`, they will be run automatically. Not required.

Reply:
Thanks, great hint! Will update in my feedstocks.